src/main/java/com/uid2/admin/vertx/service/SiteService.java

```diff
@@ -72,15 +72,41 @@ public void setupRoutes(Router router) {
             }
         }, Role.CLIENTKEY_ISSUER));
         router.post("/api/site/set-types").blockingHandler(auth.handle((ctx) -> {
-                    synchronized (writeLock) {
-                        this.handleSiteTypesSet(ctx);
-                    }
-                    }, Role.CLIENTKEY_ISSUER));
+            synchronized (writeLock) {
+                this.handleSiteTypesSet(ctx);
+            }
+        }, Role.CLIENTKEY_ISSUER));
         router.post("/api/site/domain_names").blockingHandler(auth.handle((ctx) -> {
             synchronized (writeLock) {
                 this.handleSiteDomains(ctx);
             }
         }, Role.CLIENTKEY_ISSUER));
+        router.post("/api/site/temporary/backfill_created").blockingHandler(auth.handle((ctx) -> {
```

Comment:
Why do we need an endpoint for this? Can't admin just do it on start up?

Reply:
Changed this so it is done when SiteService setupRoutes() is called and removed the unit test. Checked it still works running locally